src/pages/To.jsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react';
+import styled, { css } from 'styled-components';
+import * as F from '@/styles/fontType';
+import TextInput from '@/styles/input/TextInput';
+import PrimaryButton from '@/styles/button/PrimaryButton';
+
+const INIT_VALUE = {
+  team: '1-8',
+  name: '',
+  backgroundColor: 'begie',
+  backgroundImageURL: '',
+};
+
+export default function To() {
```

Comment:
가능하다
```suggestion
function To() {
...
}

export default To;
```
로 통일성있게 스타일을 정해주실 수 있을까요?